website_cookiebot/data/cookies_policy.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="cookies" model="ir.ui.view">
```

Comment:
This module is ment to be used instead of "website_cookie_notice", 'cause Cookiebot script manages every aspect of the cookies policy, it would be redundant to have both installed.